Kubernetes/omsagent.yaml

```diff
@@ -302,7 +302,7 @@ data:
 apiVersion: apps/v1
 kind: DaemonSet
 metadata:
-  name: omsagent
+  name: omsagent-lin
```

Comment:
we cannot rename any of the linux add-on objects as they are reconciled & tracked.

Reply:
reverted